src/pages/responsible-trading/_icon-grid.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import device from 'themes/device'
+import { GridContainer, CssGrid, CssGridColumn } from 'components/containers'
+import { Text } from 'components/elements'
+import { localize } from 'components/localization'
+// Icons
+import Demo from 'images/svg/free-demo.svg'
+import Trade from 'images/svg/money-to-lose.svg'
+import Borrowed from 'images/svg/borrowed-money.svg'
+import Limit from 'images/svg/set-limit-losses.svg'
+import Judgement from 'images/svg/bad-judgement.svg'
+import Winnings from 'images/svg/portion-of-winnings.svg'
+
+const GridCol = styled(CssGridColumn)`
+    width: 100%;
+
+    ${Text} {
+        margin-top: 1.8rem;
+
+        @media ${device.tabletL} {
+            font-size: 2rem;
+        }
+        @media ${device.tabletS} {
+            text-align: center;
+        }
+    }
+    @media ${device.tabletS} {
+        text-align: center;
+    }
+`
+const Col = ({ Icon, content }) => (
+    <GridCol>
+        <Icon />
+        <Text lh="1.55">
+            {content}
+        </Text>
+    </GridCol>
+)
+Col.propTypes = {
+    content: PropTypes.string,
+    Icon: PropTypes.func,
```

Comment:
could you use PropTypes.elementType instead of func?  It is said that it is the suitable one for react elements. What is the reason to choose func?